benchmark/pom.xml

```diff
@@ -28,6 +28,17 @@
             <version>${jmh.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.22</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.aws.greengrass</groupId>
+            <artifactId>nucleus</artifactId>
+            <version>2.6.0-SNAPSHOT</version>
```

Comment:
for benchmarking, any reason not to use a newer version?

Reply:
Just wanted to keep nucleus version consistent with what CDA uses--jmh uses CDA jar, and nucleus is a "provided" dep so we include it explicitly here